docs/UserGuide.md

```diff
@@ -152,13 +163,49 @@ at Home) to **GameBook**.
 * `add /g poker /p 0.2 /tag run-good`<br>
   Adds an entry of poker where you gained a profit of $0.20 to **GameBook** and tags the entry as "run-good".
 
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note about alerts:**<br>
+
+* If the specified DATE is in the future, an alert will be shown.
+* If an existing entry already has the same GAME_NAME and DATE, an alert will be shown.
+  * Two DATEs are the same if they fall on the exact same time (if time is specified for both), or if they fall on the same
+  day (if time is not specified for both). 
+    * Eg of DATEs regarded as same: `2020-01-01` and `2020-01-01`; `2020-01-01 10:15` and `2020-01-01 10:15`.
+    * Eg of DATEs regarded as different: `2020-01-01` and `2020-01-05`; `2020-01-01` and `2020-01-01 07:30`
+</div>
+
+
+
 ### Listing all game entries : `list`
 
 Shows a list of all game entries in **GameBook**.<br>
 
 Format:<br>
 `list`
 
+<div markdown="block" class="alert alert-info">
```

Comment:
I think this part shouldn't be here? it's in line 85

Reply:
Thanks for pointing out. Must have pasted in wrongly